creator/ingest_runs/genomic_data_loader.py

```diff
@@ -0,0 +1,397 @@
+import ast
```

Comment:
I renamed `creator/ingest_runs/manifests.py` to `creator/ingest_runs/genomic_data_loader.py` bc it seems more descriptive/intuitive.

Also wrapped most of the methods from the manifest module in a class. Class is useful so we can save access common objects more easily like the logger, the loader, etc.